app/components/dynamic-category-icon.tsx

```diff
@@ -0,0 +1,37 @@
+import {
```

Comment:
@0xRowdy moved this into `/components/ui` folder in #30. Seems good since it's our only component that's being imported by multiple routes, and others are inside their relevant folders.